cfgov/jinja2/v1/events/_macros.html

```diff
@@ -251,7 +254,10 @@ <h2 class="event-venue_heading">
             <tr>
                 <td class="{{ options.time_col_classes }}">
                     {% import 'macros/time.html' as time %}
-                    {{ time.render(bound.start_dt.render(), {'time':true, 'timezone':true}) }}
+                    {{ time.render(
+                        bound.start_dt.render(),
+                        {'time':true,'timezone':true}
```

Comment:
This overwrites the existing object entire, so it seems to make `'date': false`. I can make that explicit though for clarity